packages/uniforms-bridge-graphql/src/GraphQLBridge.ts

```diff
@@ -125,7 +125,12 @@ export default class GraphQLBridge extends Bridge {
       props.decimal = true;
     }
 
-    props.label ??= upperFirst(lowerCase(field.name));
+    if (
+      this.extras.provideDefaultLabelFromFieldName &&
+      props.label === undefined
+    ) {
+      props.label = upperFirst(lowerCase(field.name));
+    }
```

Comment:
I think we should make this bridge accept an object now. The thing is, `extras` is a map from field name to its extra props. In that case, if someone will have a `provideDefaultLabelFromFieldName` field (for some reason), it's going to break.

I propose to accept an object _plus_ this new field instead.

Reply:
Do you want it to look like this?
```ts
constructor(
  public schema: GraphQLType,
  props: {
    validator: (model: UnknownObject) => unknown,
    provideDefaultLabelFromFieldName: boolean
  }
)
```
Or do you want the `schema` to be in the object too?

Reply:
OK, I can see from the next comments that you are not sure; I would consider dividing parameters to 1. `schema` and 2. `others` if the second parameter would be unused in most cases. In this case of Graphql schema, I think the validator will be used in most cases, the same for JSONSchema, so let's change those bridges to accept one object.